index.d.ts

```diff
@@ -2032,7 +2032,15 @@ declare type Sqlite3Static = {
     installVfs: (obj: {
       io?: {
         struct: sqlite3_io_methods;
-        methods: Omit<sqlite3_io_methods, 'iVersion'>;
+        methods: { [K in keyof sqlite3_io_methods as K extends `x${string}` ? K : never]?: sqlite3_io_methods[K] };
```

Comment:
I think it's better to pick all properties that start with `x` rather than omit known static properties, specially for `sqlite_vfs` below which has a lot more of them